eclass/elisp.eclass

```diff
@@ -118,17 +117,11 @@ elisp_src_prepare() {
 		else
 			die "Cannot find ${patch}"
```

Comment:
`ELISP_PATCHES` looks like a whitespace-separated list? w/e, seems a bit out of line with modern practices

Reply:
You're right, and its docstring says "This variable is semi-deprecated, preferably use the PATCHES array instead if the EAPI supports it."

Maybe we can remove `ELISP_PATCHES` in EAPI 8.

Reply:
In fact, we can drop "if the EAPI supports it" from the comment if we're only supporting EAPI 6 and later.